policyengine_us/variables/gov/states/ri/tax/income/ri_subtractions.py

```diff
@@ -0,0 +1,12 @@
+from policyengine_us.model_api import *
+
+
+class ri_subtractions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Rhode Island AGI Subtractions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://tax.ri.gov/sites/g/files/xkgbur541/files/2022-12/2022%201041%20Schedule%20M_w.pdf"
+    defined_for = StateCode.RI
+    adds = ["ri_child_tax_rebate", "ri_social_security_modification"]
```

Comment:
make a list parameter yaml file and put all the subtractions there, use an adds function in this .py file